performance/scenarios/api-load-test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/extensions, import/no-unresolved */
+
+import http from 'k6/http';
+import { group, check, sleep } from 'k6';
+
+import { randomIntBetween, fromEnv } from '../common/utils.js';
+
+// eslint-disable-next-line import/prefer-default-export
+export const apiLoadTest = () => {
+  const path = '/participants/ecf';
```

Comment:
Just to confirm are we only testing this endpoint in the API?

Reply:
No, this is just a framework to allow us to test, we need to collaborate on what to test, what is a useful set of tests and what are appropriate thresholds.

This PR is just putting something in place to allow developers to see the effects of their code changes in case it affects performance or resilience in anyway. We need to continue and define what a performant and resilient system is.